src/middlewared/middlewared/alert/base.py

```diff
@@ -190,10 +190,18 @@ def format_alerts(product_name, hostname, node_map, alerts, gone_alerts, new_ale
         return text + "This is a test alert"
 
     if new_alerts:
-        text += "New alerts:\n" + "".join(["* %s\n" % format_alert(alert, node_map) for alert in new_alerts]) + "\n"
+        if len(gone_alerts) == 1:
+            text += "New alert"
+        else:
+            text += "New alerts"
+        text += ":\n" + "".join(["* %s\n" % format_alert(alert, node_map) for alert in new_alerts]) + "\n"
```

Comment:
Why are we still using the old Python 2 string interpolation instead of the easier to read Python3 f() function?
```suggestion
        text += ":\n" + "".join([f"* {format_alert(alert, node_map)}\n" for alert in new_alerts]) + "\n"
```

Reply:
Please focus on documentation, no python code : )